google/resource_compute_firewall_test.go

```diff
@@ -283,66 +283,6 @@ func TestAccComputeFirewall_disabled(t *testing.T) {
 	})
 }
 
-func TestAccComputeFirewall_enableLogging(t *testing.T) {
```

Comment:
What's going on here? I can't find anything pointing to this in the commit list...

Reply:
commit 47d4474294e8f88c7046e4893da9ab06e994c653 (the cherry-pick of 6a5a2757bbe075f9220ba6a50f5279d6228268d6) removes some scaffolding for tests that we didn't need over there on Earth-616, but which we do need in *this* timeline.  I have fixed it by rebasing the branch onto itself, going back in this alternate timeline, and creating a new timeline in which that removal never happened.  Someday I'll mistype a git command and be my own grandfather.